connect/tunnel_api/tunnel_api.go

```diff
@@ -462,20 +469,28 @@ func createOpenVPNClient(tunnel *Tunnel) error {
 		return fmt.Errorf("The service endpoint has no hostname or IP address associated with it.")
 	}
 
+	var proxy string
+	if tunnel.Options.ProxyHost != "" && tunnel.Options.ProxyUser != "" && tunnel.Options.ProxyPass != "" {
+		proxy = fmt.Sprintf("http-proxy %s %s /proxy.secret basic", tunnel.Options.ProxyHost, tunnel.Options.ProxyPort)
```

Comment:
Do we need to check `tunnel.Options.ProxyPort != ""` ?

Reply:
I was going to default it to 3128 if it wasn't set and I think I forgot to actually do it. Thank you for catching this! I will fix it.

Reply:
Checked and defaulted if not set now.